### PR TITLE
Fix setMigrationHandler not handling fieldConfig overrides correctly

### PR DESCRIPTION
**Pull request by academo**
setMigrationHandler was incorrectly assigning fieldConfig inside options instead of the panel object.

This PR fixes that issue and additionally removes the $$hashKey from the targets

### CODE REVIEW

File: src/lib/statusMigrationHandler.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { StatusPanelOptions } from './statusPanelOptionsBuilder';
 import { StatusThresholdOptions } from 'components/StatusThresholdOptionsEditor';
```
Comment (academo) [inline src/lib/statusMigrationHandler.ts:3]: this file ran through prettier, plenty of whitespace changes,.

**Comment by briangann** [comment]
looks like come CI needs updating (docker-compose vs docker compose)

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -1,20 +1,120 @@
-name: E2E tests
+name: CI
```
Comment (academo) [inline .github/workflows/e2e.yml:1]: Updated CI to the latest create-tool one

**Review by briangann — APPROVED** [review]
lgtm!

**Pull request merged**